python/examples/schedule/transform_a_payload_according_to_a_schedule.py

```diff
@@ -0,0 +1,120 @@
+# RUN: %PYTHON %s | FileCheck %s
+
+# A basic example of generating a payload, a schedule, and applying the latter
+# to the former. Shows how to do it from Python and from the cmd given the
+# payload and schedule are .mlir files. Run this file to see the concrete
+# schedule IR, pre-transform payload IR and transformed payload IR.
+
+import tempfile
+import subprocess
+
+from mlir.ir import Context, Location, InsertionPoint, Operation, Module
+from mlir.ir import RankedTensorType, F32Type, FloatAttr, DenseElementsAttr, UnitAttr
+from mlir.dialects import arith, func, linalg, tensor, transform
+from mlir.dialects.transform import structured
+
+
+def example_payload() -> Module:
```

Comment:
Please avoid generic names like `@example_payload` and use something descriptive instead. Specifically, what should be the name of a similar function that come next? `@example_payload_1`? How about `generate_payload_two_matmuls_and_add`? We could skip `generate_payload` if the filename was ... `generate_payload.py` or something similar ;-)